easybuild/easyconfigs/i/isoCirc/isoCirc-1.0.4-foss-2020b.eb

```diff
@@ -0,0 +1,69 @@
+easyblock = 'PythonBundle'
+
+name = 'isoCirc'
+version = '1.0.4'
+
+homepage = 'https://github.com/Xinglab/isoCirc'
+description = """isoCirc: computational pipeline to identify high-confidence
+BSJs and full-length circRNA isoforms from isoCirc long-read data
+"""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('Biopython', '1.78'),
+    ('SciPy-bundle', '2020.11'),
+    ('matplotlib', '3.3.3'),
+    ('Pysam', '0.16.0.1'),
+    ('pybedtools', '0.8.2'),
+    ('minimap2', '2.18'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('argcomplete', '1.12.3', {
+        'checksums': ['2c7dbffd8c045ea534921e63b0be6fe65e88599990d8dc408ac8c542b72a5445'],
+    }),
+    ('argh', '0.26.2', {
+        'checksums': ['e9535b8c84dc9571a48999094fda7f33e63c3f1b74f3e5f3ac0105a58405bb65'],
+    }),
+    ('crlibm', '1.0.3', {
+        'checksums': ['48e17981f90d69c6bb0013f68bacbe7a157de864a533d15dd196ca7e98348a35'],
+    }),
+    ('gffutils', '0.10.1', {
+        'checksums': ['a8fc39006d7aa353147238160640e2210b168f7849cb99896be3fc9441e351cb'],
+    }),
+    ('mappy', '2.20', {
+        'checksums': ['00c8720783ac982b514b01b86349ede9809eeaccdf1328acb5a16dfeef51458b'],
+    }),
+    ('pyfaidx', '0.6.2', {
```

Comment:
@zarybnicky Wasn't the idea to use the existing `pyfaidx` easyconfig for this? Or is a more recent version than 0.5.9.5 required?